sidebars.js

```diff
@@ -42,6 +42,7 @@ module.exports = {
       "Verifying Webhooks": [
         "receiving/verifying-payloads/why",
         "receiving/verifying-payloads/how",
+        "receiving/verifying-payloads/ngrok",
```

Comment:
It shouldn't be part of "verifying webhooks". I'd probably put it as part of "integrations" in the main (non-consuming) section of the docs.